pkg/corerp/frontend/controller/applications/testdata/graph-app-directroute-out.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "connections": [
+            {
+                "direction": "Inbound",
```

Comment:
So this is definitely the wrong direction. Is that tracked by a separate issue?

Reply:
> So this is definitely the wrong direction. Is that tracked by a separate issue?

https://github.com/radius-project/radius/issues/7038 is the tracking issue for this bug.